api/lib/api.py

```diff
@@ -0,0 +1,252 @@
+from logging import root
+from flask import Flask, request, Blueprint
+from stockfish import Stockfish
+import argparse, json, sys, logging
+
+# Local Imports
+from . import chessboard, query, utils, validation, game_utils
+from .constants import BLACK, WHITE, STOCKFISH_INVITEE_ID
+
+# BUG (see chessboard.py)
+stockfish_app = None
+
+# Other modules may load logger by using
+#  logger = logging.getLogger('some name here')
+logger = None
+
+root = Blueprint('root', __name__)
+
+@root.route('/signup', methods = ['POST'])
+def create_user():
+    args = utils.parse_args(request)
+    user_id = utils.get_first_valid_index(args, ['user'])
+    guild_id = utils.get_first_valid_index(args, ['guild'])
+    if user_id == None and guild_id == None:
+        return utils.respond(f'Cannot Sign This User Up without some Discord Identification', 400)
+    
+    if query.get_participant(user_id) != None:
+        return utils.respond(f'That user already exists Master -.o', 401)
+    
+    if query.get_participant(guild_id) != None:
+        return utils.respond(f'That user already exists Master -.o', 401)
+
+    if query.create_participant(user_id, guild_id):
+        return utils.respond(f'Created User M8!', 201)
+    
+    return utils.respond(f'UWU I couldn\'t make that for you senpai >.<', 500)
```

Comment:
>why was the text output for the bot changed (l.228)?

I had to take creative freedoms with any text outputs for behaviors not previously defined.